### PR TITLE
ponkila-ephemeral-beta: init ssvnode mainnet

### PR DESCRIPTION
**Pull request by jhvst**
Integration of https://github.com/ponkila/nixobolus/pull/46

Already deployed to prod

### CODE REVIEW

File: nixosConfigurations/ponkila-ephemeral-beta/default.nix [hunk 1]
```diff
@@ -120,9 +121,11 @@ in {
...
   # Secrets
   sops = {
-    secrets."wireguard/wg0" = {
-      sopsFile = ./secrets/default.yaml;
-    };
+    defaultSopsFile = ./secrets/default.yaml;
+    secrets."wireguard/wg0" = {};
+    secrets."ssvnode/password" = {};
+    secrets."ssvnode/publicKey" = {};
+    secrets."ssvnode/privateKey" = {};
```
Comment (tupakkatapa) [inline nixosConfigurations/ponkila-ephemeral-beta/default.nix:128]: Nice to see the usage of defaultSopsFile if it is actually working.

Why are these here even though sops isn't actually used?

**Comment by tupakkatapa** [comment]
LGTM otherwise.

File: nixosConfigurations/ponkila-ephemeral-beta/default.nix (hunk 1 above)
Comment (jhvst) [reply nixosConfigurations/ponkila-ephemeral-beta/default.nix:128]: Main idea was to indeed see if it works, and it does. I now updated the config to actually use these values. One upside in general to configure these this way is to ensure that the keys do not get accidentally lost or overridden, given that the keys are found from git now.
